dev/ProgressRing/TestUI/ProgressRingStoryboardAnimationPage.xaml

```diff
@@ -34,23 +34,9 @@
                             </Grid.Resources>
 
                             <VisualStateManager.VisualStateGroups>
-                                <VisualStateGroup x:Name="SizeStates">
```

Comment:
can we keep the old names and just add new ones? Changing the state names will break if someone has retemplated the control and has the old template.

Reply:
Done - There is now Active, InActive, and DeterminateActive (the new state).